### ISSUE TITLE
Mailbox Simulator support

### ISSUE THREAD

**Issue by WhyNotHugo** [django-ses/django-ses#197]
I'd love to see support for using AWS's [Mailbox Simulator](Mailbox Simulator). I'd set this up for my staging and development environments.

What do you think of a flag `SES_USE_SIMULATOR`? The default value would be `False`, to avoid breaking any existing usages.

**Comment by pcraciunoiu** [comment]
Yeah I'd be open to it if you want to make a PR. Sounds like you'd be able to test it in a live environment, so that's great.
